.circleci/config.yml

```diff
@@ -39,6 +39,15 @@ commands:
               cd charm-v$CHARM_VER
               ./build charm++ netlrts-linux-x86_64 -j4 --with-production
             fi
+            # prepare testing dependencies
+            python3 -m venv $HOME/venv
+            source $HOME/venv/bin/activate
+            pip install --upgrade pip
+            pip install --upgrade wheel
+            pip install --upgrade setuptools
+            # matplotlib 3.3.0 breaks yt 3.6.0 (latest stable version at this
```

Comment:
yt 3.6.1 was released just a couple days ago, so we should no longer need this. That said, experience has shown it's probably wise to continue to pin the matplotlib version.